app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -434,6 +441,11 @@ class CommonTriageQuestionsController @Inject() (
       }
     }
 
+  def amendWhoAreYouSubmittingFor(): Action[AnyContent] =
```

Comment:
getting a small unit test to check things like the title and the back link location of this page would be good